releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,17 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.2.0 - 
+
+### New Features
+* Added support for feature flags, which allows feature flag settings to be saved to a Kubernetes ConfigMap.

```

Comment:
This depends on the release of FM libraries of other languages. We will need some coordination before we can declare the support for feature flags.

Reply:
I see. So the release sequence should be FM libraries (dotnet/java spring) -> k8s provider.

Reply:
Yes, something like that, and in k8s provider release, we will say which version of FM library people need to use and on which platform. We will need quickstart/tutorial docs for k8s provider FM as well.